### PR TITLE
Fiche pci and others

### PR DESCRIPTION
**Pull request by H4516022Y**
hello team,

minor corrections for old docs and a new publication pour PCI 

thanks for your review

### CODE REVIEW

File: pages/account_and_service_management/account_information/security-specification-pci/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+| **Name** | **Capacity** | **Documentation** |
+| --- | --- | --- |
+| Control Panel and service | Manage customer accounts and services on which each account has access rights. | [Preparing an environment for using the OpenStack API](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api)<br>[Getting started with the OpenStack API](/pages/public_cloud/compute/starting_with_nova)<br>[API Rate Limits](/pages/public_cloud/compute/api_rate_limits) |
+
+### 7. Accounts - User
+
+#### 7.1 Control plane
+
+Using your customer account on the OVHcloud Control Panel, you are able to manage your service using [three different contacts](/pages/account_and_service_management/account_information/managing_contacts).<br>
+OVHcloud uses another account with an internal NIC to refer a customer having subscribed to several services.
```
Comment (Jessica41) [inline pages/account_and_service_management/account_information/security-specification-pci/guide.en-gb.md:74]: Hello, @H4516022Y I do not know if internal NIC is an appropriate term to use here. I am not sure to understand the statement.

Comment (H4516022Y) [reply pages/account_and_service_management/account_information/security-specification-pci/guide.en-gb.md:74]: hello, Yes it is and it's for internal use to identify any customer having several NICs and services

File: pages/account_and_service_management/account_information/security-specification-pci/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+id: ec84433e-5187-4642-8e73-310292eaf82e
+full_slug: Public-cloud-instance-security-overview
```
Comment (OVH-Team-Guides) [inline pages/account_and_service_management/account_information/security-specification-pci/meta.yaml]:
```suggestion
full_slug: public-cloud-instance-security-overview
```
Upper case is not a good practice for SEO

File: pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.fr-fr.md [hunk 3]
```diff
@@ -46,7 +46,9 @@ Reprise des SLA des conditions particulières par composante du service.
...
+| L'ensemble du service | Le taux de disponibilité mensuel est de 99,95 % pour l'ensemble du service | Nombre total  de minutes du mois considéré, déduction faite du nombre de minutes d’indisponibilité du mois concerné, le tout divisé par le nombre total de minutes du mois considéré. Pour le calcul des dédommagements, le temps d’indisponibilité est calculé à partir de l’ouverture du ticket incident, jusqu'à la résolution du dysfonctionnement. | 1- si de taux disponibilité mensuel < 99,95 %, crédit de 10% du prix du service impacté. <br>2- si taux de disponibilité mensuel <99%,9 crédit de 30% du prix du service impacté |
```
Comment (OVH-Team-Guides) [inline pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.fr-fr.md]:
```suggestion
| L'ensemble du service | Le taux de disponibilité mensuel est de 99,95 % pour l'ensemble du service | Nombre total de minutes du mois considéré, déduction faite du nombre de minutes d’indisponibilité du mois concerné, le tout divisé par le nombre total de minutes du mois considéré. Pour le calcul des dédommagements, le temps d’indisponibilité est calculé à partir de l’ouverture du ticket incident, jusqu'à la résolution du dysfonctionnement. | 1. si le taux disponibilité mensuel est < 99,95%, crédit de 10% du prix du service impacté. <br>2. Si le taux de disponibilité mensuel est < 99,99%, crédit de 30% du prix du service impacté |
```

File: pages/account_and_service_management/account_information/security-specification-privatecloud-VMware/guide.fr-fr.md [hunk 4]
```diff
@@ -46,7 +46,9 @@ Reprise des SLA des conditions particulières par composante du service.
...
+| L'ensemble du service | Le taux de disponibilité mensuel est de 99,95 % pour l'ensemble du service | Nombre total  de minutes du mois considéré, déduction faite du nombre de minutes d’indisponibilité du mois concerné, le tout divisé par le nombre total de minutes du mois considéré. Pour le calcul des dédommagements, le temps d’indisponibilité est calculé à partir de l’ouverture du ticket incident, jusqu'à la résolution du dysfonctionnement. | 1- si taux de disponibilité mensuel < 99,95 %, crédit de 10% du prix du service impacté. <br>2- si taux de disponibilité mensuel <99%,9 crédit de 30% du prix du service impacté |
```
Comment (OVH-Team-Guides) [inline pages/account_and_service_management/account_information/security-specification-privatecloud-VMware/guide.fr-fr.md]:
```suggestion
| L'ensemble du service | Le taux de disponibilité mensuel est de 99,95 % pour l'ensemble du service | Nombre total de minutes du mois considéré, déduction faite du nombre de minutes d’indisponibilité du mois concerné, le tout divisé par le nombre total de minutes du mois considéré. Pour le calcul des dédommagements, le temps d’indisponibilité est calculé à partir de l’ouverture du ticket incident, jusqu'à la résolution du dysfonctionnement. | 1. Si le taux de disponibilité mensuel est < 99,95 %, crédit de 10% du prix du service impacté. <br>2. Si le taux de disponibilité mensuel est < 99,99%, crédit de 30% du prix du service impacté |
```

File: pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.fr-fr.md (hunk 3 above)
Comment (OVH-Team-Guides) [reply pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.fr-fr.md]: @H4516022Y , please check if these suggestions are OK.

**Comment by H4516022Y** [comment]
yes , everything is OK :)

**Pull request merged**